script.module.ssearlo/addon.xml

```diff
@@ -0,0 +1,21 @@
+<?xml version="1.0" encoding="UTF-8" standalone="yes"?>
+<addon id="script.module.ssearlo"
+       name="ssearlo"
+       version="0.0.22"
+       provider-name="Al DAmico">
+  <requires>
+    <import addon="xbmc.python" version="2.26.0"/>
+    <import addon="script.module.requests" version="2.9"/>
+    <import addon="script.module.six" version="1.11.0"/>
+  </requires>
+  <extension point="xbmc.python.module" library="lib" />
+  <extension point="xbmc.addon.metadata">
+    <summary lang="en_GB">Python client library for reading Server Sent Event streams based on sseclient.</summary>
+    <description lang="en_GB">Cloned from from https://github.com/btubbs/sseclient 0.0.22 due to bug in 0.0.24</description>
+    <platform>all</platform>
+    <license>MIT License</license>
```

Comment:
MIT please

Reply:
(spdx identifier)

Reply:
Sorry for my ignorance, but I'm not sure what your comments mean.  As required, there is a LICENSE file with the MIT text in it (cloned from the original author), and I'm not sure what spdx identifier is...

Reply:
I mean to replace the "MIT License" text with just "MIT". It's the spdx identifier for the license: https://spdx.org/licenses/